ptero_lsf/implementation/celery_tasks/lsf_task.py

```diff
@@ -63,12 +63,15 @@ def run(self, job_id, lsf_job_id):
             backend = celery.current_app.factory.create_backend()
             backend.kill_lsf_job(lsf_job_id)
         except Exception as exc:
-            delay = DELAYS[self.request.retries]
-            LOG.exception("Exception while trying to kill job (%s) with lsf "
-                    "job id (%s) retrying in %s seconds.  Attempt %d of %d.",
-                    job_id, lsf_job_id, delay, self.request.retries + 1,
-                    self.max_retries + 1, extra={'jobId': job_id})
-            self.retry(exc=exc, countdown=delay)
+            if "Job has already finished" in exc.message:
```

Comment:
Is the Exception that comes back from LSF typed somehow? Could we catch a specific exception rather than string matching?